packages/toolpad-app/src/toolpad/AppEditor/PageEditor/PageOptionsPanel.tsx

```diff
@@ -122,22 +125,28 @@ export default function PageOptionsPanel() {
       {FEATURE_FLAG_AUTHORIZATION ? (
         <div>
           <Typography variant="body2">Authorization:</Typography>
-          <FormControlLabel
-            control={<Checkbox checked={allowAll} onChange={handleAllowAllChange} />}
-            label="Allow access to all roles"
-          />
-          <Autocomplete
-            multiple
-            options={Array.from(availableRoles.keys())}
-            value={allowAll ? [] : allowedRoles}
-            onChange={handleAllowedRolesChange}
-            disabled={allowAll}
-            fullWidth
-            noOptionsText="No roles defined"
-            renderInput={(params) => (
-              <TextField {...params} label="Allowed roles" placeholder="Roles" />
-            )}
-          />
+          {isPaidPlan ? (
```

Comment:
Would the upgrade message always appear to anyone who is using the free plan?
If so I guess it's better to just hide the controls here as this panel is always open?

Reply:
Agreed with the fact that it could be a little disruptive and annoying. To me its okay to advertise the fact that there is an upgrade possible as much as we can to enable feature discovery - so we can potentially keep it for now